### ISSUE TITLE
Code for collecting replies and commets？

### ISSUE THREAD

**Issue by srtianxia** [KaiDMML/FakeNewsNet#10]
Hi,
Great job, thanks! But can you provide the code for collecting some of the dimensions of social engagments including replies and commets?
Thanks a lot !

**Comment by SaschaStenger** [comment]
I would be interested in this as well.
I tried contacting you, as it was said in the paper, that you only share this code upon request. But so far i did not get an answer to my mail, but maybe i took the wrong mail adress.

**Comment by rlleshi** [comment]
I would also be interested on this. Were you guys able to contact him?

**Comment by SaschaStenger** [comment]
No, but i'm currently thinking about modifying the download script anyways. 
The [twarc](https://github.com/DocNow/twarc/blob/master/twarc/client.py#L564) library offers a method for getting replies and this could be integrated.

**Comment by mdepak** [comment]
Sorry for the delayed response. The code for collecting social engagements that use the web scraping is broken with the latest twitter UI changes. @SaschaStenger Please feel free to integrate your changes for downloading engagements with `FakeNewsNet` and send a pull request.
